### PR TITLE
added function to update state on successful video submission

### PR DESCRIPTION
**Pull request by seangtkelley**

### CODE REVIEW

**Comment by aasnani** [comment]
Don't forget to **update yo state**! - *Gandhi, 2019, colorized*

File: src/ContentPage.jsx [hunk 1]
```diff
@@ -183,10 +184,11 @@ class ContentPage extends React.Component {
   constructor() {
     super();
 
-    const dummyWikiID = 162393; // this is going to change when we use routers
+    const dummyWikiID = 1424309; // this is going to change when we use routers
```
Comment (sjakati98) [inline src/ContentPage.jsx:187]: These numbers seem like a code. The numbers, WHAT DO THEY MEAN?

File: src/ContentPage.jsx [hunk 2]
```diff
@@ -183,10 +184,11 @@ class ContentPage extends React.Component {
...
     
     this.loadWikiData = this.loadWikiData.bind(this);
     this.loadAbaxaciData = this.loadAbaxaciData.bind(this);
+    this.addVideo = this.addVideo.bind(this);
```
Comment (sjakati98) [inline src/ContentPage.jsx:191]: Are you sure it's not that

File: src/ContentPage.jsx (hunk 1 above)
Comment (aasnani) [reply src/ContentPage.jsx:187]: **MASON!!!!!!!!!!!!!**

**Review by sjakati98 — COMMENTED** [review]
approved.

**Pull request merged**